pkg/apis/scheduling/v1beta1/labels.go

```diff
@@ -42,6 +42,10 @@ const QueueNameAnnotationKey = GroupName + "/queue-name"
 // PodPreemptable is the key of preemptable
 const PodPreemptable = "volcano.sh/preemptable"
 
+// PodPreemptStableTime is the key of preempt-stable-time, value's format "600s","10m" 
+// Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h". 
+const PodPreemptStableTime = "volcano.sh/preempt-stable-time"
```

Comment:
Whether job need a prempt-stable-time as well. If needs, it seems using `PreemptStableTime` is more reasonable.

Reply:
yes! already updated